encryption.md

```diff
@@ -1,13 +1,8 @@
 # Message Encryption
 
-DIDComm supports two types of message encryption: Authenticated Sender Encryption and Anonymous Sender Encryption. Both forms are encrypted to the recipient, but only Authenticated Sender Encryption provides assurances of who the sender is.
-
 The encrypted form of a JWM is a JWE. The JOSE family defines [JSON Web Algorithms](https://tools.ietf.org/html/rfc7518) (JWAs) which standardize certain cryptographic operations that are related to preparing JOSE structures. For the purposes of interoperability, DIDComm messaging does not support all JWAs, rather it takes a subset of the supported algorithms that are applicable for the following cases around secure messaging. These supported algorithms are listed here.
 
-
-## Sender Authenticated Encryption
-
-When a sender would like to encrypt a message for a recipient or multiple recipients and also be authenticated such that the recipients can determine they were the party that encrypted the message, the JWA of `ECDH-1PU` defined by [draft](https://tools.ietf.org/html/draft-madden-jose-ecdh-1pu-03) SHOULD be used within the structure of a JWE.
+For an encrypted DIDComm Message, the JWA of `ECDH-1PU` defined by [draft](https://tools.ietf.org/html/draft-madden-jose-ecdh-1pu-03) SHOULD be used within the structure of a JWE.
```

Comment:
Could we upgrade this to a MUST rather than a SHOULD now that ECDH-1PU is the only supported JWA? I believe originally this was only a SHOULD rather than a MUST because we supported the use of ECDH-ES, but with that gone, I think it would be more useful for us to upgrade this to a MUST.

As a side note, we're going to need to make sure this draft makes it into a WG as well. Would love for others to help assist with that effort if anyone has contacts at IETF that may be able to help with that.

Reply:
changed SHOULD to MUST. Great point.